src/app/helptext/storage/volumes/manager/manager.ts

```diff
@@ -21,6 +21,8 @@ manager_disknumExtendConfirmMessage : T("It is not recommended to extend a pool
 
 manager_vdevtypeErrorMessage : T("Adding data vdevs of different types is not supported."),
 
+manager_specialVdevTypeErrorMessage : T("Metadata vdev should be of the same type as the data vdevs."),
```

Comment:
Is this a recommendation or a hard error? Rewording to to be a hard error
```suggestion
manager_specialVdevTypeErrorMessage : T("Metadata vdev must be the same type as the data vdevs."),
```